### DISCUSSION TITLE
Can a release be cut?

### DISCUSSION THREAD

**Discussion by kieranm** [pacocoursey/cmdk#155] [General]
Hey all,

A number of useful bug fixes and accessibility fixes have been merged, but a release has not been put out to npm since March, so we can't really use any of them. When will the next release be cut? Is it possible to accelerate this?

Thanks a lot
Kieran

**Comment by pacocoursey** [comment]
Released: https://github.com/pacocoursey/cmdk/releases/tag/v0.2.1
